armory/datasets/context.py

```diff
@@ -0,0 +1,16 @@
+import tensorflow as tf
+
+
+class VideoContext:
+    def __init__(self, x_shape, frame_rate):
+        self.x_shape = x_shape
```

Comment:
do we expect `x_shape` and `input_type` to ever actually be used? If not, probably clearer to remove so in the future we're not wondering where/if it's used

Reply:
Yes, they are both used in the preprocessor here: https://github.com/twosixlabs/armory/blob/f07ae034e7a4415bf0bfeb3c543133bdf4e7b194/armory/datasets/preprocessing.py#L84-L89

Reply:
I think we should make preprocessing as simple as possible by not feeding in the context object (and removing the corresponding assertions). I'd think the context should only be needed for grabbing frame_rate when the exporter is instantiated

Reply:
Addressed in 18ce134d72cb1318a7a3489b827a98e687945530. Without the input validation should the preprocessor function still be split into two functions to handle labels separately or can I merge these now?